### PR TITLE
Consistent training and test sets + removes tf-torch differences

### PR DESCRIPTION
**Pull request by ricvolpe**

### CODE REVIEW

**Review by wichersn — CHANGES_REQUESTED** [review]

File: extractors/configs/hptuning_config_tf_baseline.yaml [hunk 1]
```diff
@@ -11,24 +11,29 @@ trainingInput:
...
+        categoricalValues:
+        - "False"
 
       - parameterName: gin_bindings=tf_extractor.cnn_from_obs.learning_rate
         type: DOUBLE
         minValue: .001
         maxValue: .1
         scaleType: UNIT_LOG_SCALE
-
-      - parameterName: gin_bindings=tf_extractor.cnn_from_obs.reg_amount
```
[the remark this answers is not in the document]
Comment (ricvolpe) [reply extractors/configs/hptuning_config_tf_baseline.yaml:21]: I am removing it because the Torch code does not have it and I understood that we decided to remove all differences between TF and Torch in favour of Torch.
Also, it did not seem very useful when I looked at the values that led to good perfomance. For example this is an analysis of regularization values (x-axis) and test auc (y-axis) for the fine tuning technique in TF:
[
Screen Shot 2020-12-26 at 10 36 13
](url)

File: extractors/configs/hptuning_config_torch_baseline.yaml [hunk 2]
```diff
@@ -17,12 +17,12 @@ trainingInput:
         minValue: 0.001
         maxValue: 0.1
         scaleType: UNIT_LOG_SCALE
-
-      - parameterName: gin_bindings=TorchExtractor.weight_decay
```
[the remark this answers is not in the document]
Comment (ricvolpe) [reply extractors/configs/hptuning_config_torch_baseline.yaml:21]: I looked into this and weight decay seemed not useful at all. See below an analysis of weight decay values (x-axis) and test auc (y-axis) for both the subnetwork (above) and torch baseline (below) techniques.
Screen Shot 2020-12-26 at 10 37 15

What do you think?

**Review by wichersn — APPROVED** [review]
